lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -248,263 +351,12 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
     return create_bundle_resource(
         bundle_id = str(uuid4()),
         timestamp = datetime.now().astimezone().isoformat(),
-        source = f"{REDCAP_URL}{project_id}/{record_id}",
+        source = f"{REDCAP_URL}{str(redcap_record_instance.project.id)}/{redcap_record_instance.id}",
```

Comment:
This is outside the for loop so `redcap_record_instance` does not exist, but you can use `enrollment` here to get project id and record id. 

Also side question, I thought the `str` conversion isn't needed for f-strings?

Reply:
Nice catch, there. When I saw your comment I was all "but it worked". Ah, python's scoping ... redcap_record_instance is available outside the loop. I switched to enrollment because that is saner.

In the second commit I'm removing the str() from the f'strings.